### PR TITLE
Update settings docs for IMU decimation [ESD-1014]

### PR DESCRIPTION
**Pull request by cbeighley**
Also update miscellaneous other out of date settings.

@silverjam @dzollo @nicolekelley

### CODE REVIEW

File: piksi_tools/console/settings.yaml [hunk 1]
```diff
@@ -2564,7 +2566,7 @@
   expert: true
   type: string
   units: N/A
-  default value: '72,74,117,522,527'
+  default value: '23,65,72,74,81,97,117,134,136,137,138,139,144,149,163,165,166,167,171,175,181,185,187,188,189,190,257,258,259,520,522,524,526,527,528,1025,2304,2305,2306,30583,65280,65282,65535'
```
Comment (silverjam) [inline piksi_tools/console/settings.yaml]: If these are the CAN ports (reviewing from mobile), I don’t think this is correct.  The CAN ports had an intentionally much more restrictive set of defaults.

File: piksi_tools/console/settings.yaml [hunk 2]
```diff
@@ -824,7 +824,7 @@
   expert: false
   type: string
   units: N/A
-  default value: '23,65,72,74,81,97,117,134,136,137,138,139, 144,149,163,165,166,167,171,175,181, 185,187,188,189,190,257,258,259,520, 522,524,526,527,528,1025,2304,2305, 2306,30583,65280,65282,65535'
+  default value: '23,65,72,74,81,97,117,134,136,137,138,139,144,149,163,165,166,167,171,175,181,185,187,188,189,190,257,258,259,520,522,524,526,527,528,1025,2304/50,2305,2306/50,30583,65280,65282,65535'
```
[the remark this answers is not in the document]
Comment (cbeighley) [reply piksi_tools/console/settings.yaml]: Will add.

File: piksi_tools/console/settings.yaml (hunk 1 above)
Comment (cbeighley) [reply piksi_tools/console/settings.yaml]: I took these from https://github.com/swift-nav/piksi_buildroot/blob/master/package/ports_daemon/ports_daemon/src/whitelists.c, I can just remove from this PR if you prefer.

Comment (silverjam) [reply piksi_tools/console/settings.yaml]: Ok, either way works, that's much more enabled messages than I expected it to have.

Comment (cbeighley) [reply piksi_tools/console/settings.yaml]: I'll remove for now.

**Pull request merged**